docs/product-manuals/tasklist/deployment/api-client.md

```diff
@@ -6,37 +6,41 @@ description: "Let's take a closer look at the endpoint, authentication, access t
 
 ## Endpoint
 
-Tasklist provides an GraphQL API at endpoint `/graphql`.
+Tasklist provides a GraphQL API at endpoint `/graphql`.
 
 ## Authentication
 
-To access the API endpoint you need an access token. 
-Your client needs to send a header in each request: 
+To access the API endpoint, you need an access token. 
+
+Your client must send a header in each request: 
 
 `Authorization: Bearer <Token>`
 
-For example send a request by using *curl*:
+For example, send a request using *curl*:
+
+[//]:# (How is curl normally written? Curl? or cURL, or CURL?)
```

Comment:
Yes, all of the above. 😆 

cURL is the product or tool, curl is operation you do with it. A great item for the style guide. 

I would recommend we standardize on curl, and consider `curl` to make it visually stand out (instead of italics or bold).